osmaxx/conversion/converters/utils.py

```diff
@@ -25,3 +34,11 @@ def zip_folders_relative(folder_list, zip_out_file_path=None):
         finally:
             os.chdir(old_dir)
     return zip_out_file_path
+
+
+def recursive_getsize(path):
+    size = 0
+    for entry in scandir(path):
+        if not entry.name.startswith('.') and entry.is_file():
+            size += os.path.getsize(entry.path)
+    return size
```

Comment:
I had something similar in the beginning, but switched to this because of readability reasons - but I don't really care which version should get in, of you think the `sum()` is more explicit/readable, I'll include that one.